web/src/components/messages/error.tsx

```diff
@@ -99,21 +104,27 @@ export const Error: React.FC<ErrorProps> = ({ title, error, isLokiRelated }) =>
   }, [error]);
 
   React.useEffect(() => {
-    //jest crashing on getLokiReady not defined so we need to ensure the function is defined here
-    if (getLokiReady && isLokiRelated) {
-      getLokiReady()
-        .then(() => {
-          setReady(true);
-        })
-        .catch(err => {
-          console.error(getHTTPErrorDetails(err));
-          setReady(false);
-        })
-        .finally(() => {
-          setLoading(false);
-        });
+    //jest crashing on getStatus not defined so we need to ensure the function is defined here
+    if (!getStatus) {
+      return;
     }
-  }, [isLokiRelated]);
+
+    getStatus(ContextSingleton.getForcedNamespace())
+      .then(status => {
+        console.info('status result', status);
```

Comment:
debug log to remove?

Reply:
I feel it's useful to keep that in the console in case the UI don't show everything but I can remove it if you prefer